tests/test_configs_work.py

```diff
@@ -112,3 +112,38 @@ def test_configs_work_in_nbqa_ini(
     expected_err = ""
     assert sorted(out.splitlines()) == sorted(expected_out.splitlines())
     assert sorted(err.splitlines()) == sorted(expected_err.splitlines())
+
+
+def test_setupcfg_is_preserved(capsys: "CaptureFixture") -> None:
+    """
+    Check setup.cfg file is automatically picked up by nbqa.
+
+    Parameters
+    ----------
+    capsys
+        Pytest fixture to capture stdout and stderr.
+    """
+    with open("setup.cfg", "w") as handle:
+        handle.write(
+            dedent(
+                """\
+            [flake8]
+            ignore=F401
+            select=E303
+            quiet=1
+            """
+            )
+        )
```

Comment:
Can be simplified to:
```python
Path("setup.cfg").write_text("[flake8]\nignore=F401\nselect=E303\nquiet=1")
```

Reply:
@s-weigand I find the `dedent`version more readable compared to the your simplified code snippet. But I agree to use `Path.write_text`. So I suggest, `Path.write_text(dedent(...))`

Reply:
I use [MagicPython](https://github.com/MagicStack/MagicPython) for syntax highlighting, which also works on strings, so for me it is quite readable. But yeah, if is more than 2 lines, than the decent might be more readable.